discord/interactions/connectCommand.ts

```diff
@@ -147,9 +179,14 @@ export const handleConnectCommand = async (
         reconnectButtons
       );
 
+      const isEthereum = isEthereumNetwork(
+        alreadyDiscordMembers[0].starknetNetwork
+      );
+
       await interaction.reply({
-        content:
-          "You have already linked a Starknet wallet to this Discord server on all networks. Use `/starky-disconnect` first if you want to link a new one",
+        content: `You have already linked a ${
```

Comment:
No need to have dynamic text here. Simply write "You have already linked a wallet to this Discord server on all networks. Use `/starky-disconnect` first if you want to link a new one"